lightly/api/api_workflow_datasets.py

```diff
@@ -100,13 +104,23 @@ def create_dataset(self, dataset_name: str, dataset_type: str = None):
             >>> client.create_dataset('your-dataset-name', dataset_type=DatasetType.VIDEOS)
```

Comment:
Since we refactor this maybe also change this here to `dataset_id = client.create_dataset('your-dataset-name', dataset_type=DatasetType.VIDEOS)`

Reply:
`create_dataset` does not return the new dataset id. The dataset id is set on the client:
```
client.create_dataset('your-dataset-name', dataset_type=DatasetType.VIDEOS)
dataset_id = client.dataset_id
```

Reply:
I'll add some more explanations about this to the docstring.

Reply:
You're right! My fault